qso/handlers.go

```diff
@@ -50,6 +74,18 @@ func (handler *QsoHandler) Create(c *fiber.Ctx) error {
 	return c.JSON(item)
 }
 
+// Update
+//	@Summary	Update QSO record by ID
+//	@Tags		QSO
+//	@Accept		json
+//	@Produce	json
+//	@Param		id		path		int	true	"QSO ID"
+//	@Param		request	body		Qso	true	"QSO data"
+//	@Success	200		{object}	Qso
+//	@Failure	400		{object}	object
+//	@Failure	404		{object}	object
+//	@Failure	500		{object}	object
+//	@Router		/api/qso/{id} [put]
 func (handler *QsoHandler) Update(c *fiber.Ctx) error {
```

Comment:
Same questions I had for the "POST" method apply here, although the ID actually needs to be displayed.